editors/code/src/spUtils.ts

```diff
@@ -1,11 +1,33 @@
 ﻿import * as vscode from "vscode";
+import path from "path";
+import { getCtxFromUri, lastActiveEditor } from "./spIndex";
+import { ProjectMainPathParams, projectMainPath } from "./lsp_ext";
+import { URI } from "vscode-uri";
 
 export function sleep(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
-export function isSPFile(fileName: string) {
-  return /(?:\.sp|\.inc)\s*^/.test(fileName);
+export function isSPFile(filePath: string): boolean {
+  return /\.(sp|inc)$/i.test(filePath);
```

Comment:
I don't think the compiler is case insensitive?

Reply:
Hm I think I changed that regex because it wasn't working, I tested it in regex101.com but it wasn't matching with anything that function was receiving, but I could be wrong.

Reply:
The trailing `^` was 100% wrong, that's for sure, but you can remove the `i` in the new one.

Reply:
```suggestion
  return /\.(sp|inc)$/.test(filePath);
```